sql/2021/pwa/manifests_preferring_native_apps_sw.sql

```diff
@@ -0,0 +1,35 @@
+#standardSQL
+# % manifests preferring native apps for service worker pages
```

Comment:
Use this in the "Installability" section, where we'll also count usage of the `beforeinstallprompt` event.